README.md

```
@@ -41,25 +58,68 @@ For detailed documentation on the DCC project, please refer to the [DCC Document
 ```
-API_PORT=8000
-API_URL=http://localhost:${API_PORT}
 
-FEEDBACK_GITHUB_TOKEN=TODO
+#### Optional Environment Variables
+
+The following environment variables have defaults and can be overridden as needed:
+
+| Variable | Description | Default | Type |
+|----------|-------------|---------|------|
+| **App Configuration** |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 4; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)